ParisBNF/abb/BNFabb39.xml

```diff
@@ -721,16 +725,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                                                                                   <ref type="mss" corresp="BNFet149"/>,<locus from="28vb" to="29va"/>.</note>
                                                                                                 </msItem>
 
-                                                                <msItem xml:id="p1_i5.29">
+                                                                <msItem xml:id="p1_i9.18">
                                                             <locus from="203vb" to="204ra"/>
-                                                      <title type="complete">Lexicon for <ref type="work" corresp="LIT2194Prophe">the Books of the Prophets</ref></title>
-                                                          <textLang mainLang="gez" otherLangs="am">
-                                                          </textLang>
+                                                      <title type="complete">Lexicon for the Books of the Prophets</title>
+                                                                  <textLang mainLang="gez" otherLangs="am"/>
                                                     <incipit xml:lang="gez"><hi rend="rubr">እጽሕፍ፡ </hi>ነገረ፡ ነቢያት፤ በመስልስት፡ ዘ፡ በበ፫፡ ሰብእ፡ በ፩፡ ፈረስ፡ ምንዙኅ፡ ዘ፡ ሕንቅቅ፡                                                                                                       </incipit>
-                                                    </msItem>    </msItem>
-
-
-
+                                                    </msItem>   
+                            </msItem>   
+                    </msItem>   
```

Comment:
when you added `</msItem> `above take one out here

Reply:
as I wrote - you must close` /msItem` above in line 377 and remove one extra one after p1_i9 in line 737 or 738